spec/chewy/rake_helper_spec.rb

```diff
@@ -502,4 +529,45 @@
       end
     end
   end
+
+  describe '.delete_by_query_options_from_env' do
+    subject(:options) { described_class.delete_by_query_options_from_env(env) }
+    let(:env) do
+      {
+        'WAIT_FOR_COMPLETION' => 'false',
+        'REQUESTS_PER_SECOND' => '10',
```

Comment:
`REQUESTS_PER_SECOND` is defined as a float, so why do you use an integer in the example? 
Why not: `'REQUESTS_PER_SECOND' => '10.0',` ?